primevue-gen/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "primvue-gen",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
There isn't any `index.js` file, and this still works just fine. I wonder if we could just remove this line.

Reply:
I see you have addressed this commit with 43381cccbaa8ee644d3c72c043c22c552fec6586.

This conversation is now resolved.